repo/packages/H/hello-world/6/package.json

```diff
@@ -5,6 +5,7 @@
   "minDcosReleaseVersion": "1.9",
   "name": "hello-world",
   "packagingVersion": "3.0",
+  "preInstallNotes": "This DC/OS Service is currently in preview.",
```

Comment:
This should be updated in the `dcos-commons/frameworks/helloworld/universe` location, as well.